cli/test/test_scaffold.py

```diff
@@ -0,0 +1,29 @@
+from click.testing import CliRunner
+
+import popper.commands.cmd_scaffold as scaffold
+import popper.commands.cmd_run as run
+from test_common import PopperTest
+
+import unittest
+import os
+
+
+class TestScaffold(PopperTest):
+
+    def test_scaffold(self):
+
+        repo = self.mk_repo()
+        runner = CliRunner()
+        file_loc = repo.working_dir+'/wf.yml'
```

Comment:
`mk_repo` creates a git repo. In this case, we don't really need one, so we can instead use `tempfile`:

```python
wf_dir = tempfile.mkdtemp()
file_loc = f'{wf_dir}/wf.yml'
```

BTW, we can use f-strings here as well as shown above, in order to keep it consistent with the rest of the codebase